.github/workflows/CI.yml

```diff
@@ -18,7 +18,6 @@ jobs:
       fail-fast: false
       matrix:
         version:
-          - '1.6'
```

Comment:
@mtfishman just realized about this change, I feel like we should still keep 1.6?

Reply:
Yes, we should keep 1.6.

Reply:
Ok, but wouldn't this break the tests then?
https://github.com/mtfishman/ITensorChemistry.jl/blob/c381b58967f37db47519bd275d1da3b85055b32f/src/molecular_orbital_hamiltonian.jl#L32

Reply:
I have lost track of which notation is specific to 1.6 vs. 1.7, but we should make everything compatible with 1.6. We don't have to do that in this PR though, just a general goal. I was being lax about it since this was just an experimental package for the time being.